tests/phpunit/PaymentMethods/WC_Stripe_UPE_Display_Order_Fee_Test.php

```
@@ -0,0 +1,120 @@
+		$order_helper = WC_Stripe_Order_Helper::get_instance();
+		$order_helper->update_stripe_fee( $order, '0.59' );
+		$order_helper->update_stripe_currency( $order, 'USD' );
+		$order->save();
+
+		ob_start();
+		$this->gateway->display_order_fee( $order->get_id() );
+		$output = ob_get_clean();
+
+		// No raw currency code appended when currencies match.
+		$this->assertStringNotContainsString( ' USD', $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_display_order_fee_same_currency_no_code_appended is not named in camelCase. (undefined)

(CamelCaseMethodName)